.gitignore

```diff
@@ -41,6 +41,7 @@ app/style.css.map
 dist
 main.js
 main.js.map
+tsconfig.json
```

Comment:
I think this will affect all `tsconfig.json` files, make sure we ignore only the  one in the root, I think is `/tsconfig.json`